.gitignore

```diff
@@ -28,9 +22,7 @@ _tmp
 /client-gen
 /lister-gen
 /informer-gen
-/metering-custom.yaml
```

Comment:
I feel like we do still want this gitignored. It's a common name for the CR and the CR changes fairly often.